data/sql/updates/pending_db_characters/rev_1617907126348389400.sql

```diff
@@ -19,4 +19,11 @@ ALTER TABLE `auctionhouse` CHANGE `auctioneerguid` `houseid` TINYINT(3)  UNSIGNE
 
 ALTER TABLE `characters` CHANGE `transguid` `transguid` MEDIUMINT DEFAULT 0 NOT NULL;
 
-ALTER TABLE `groups` CHANGE `icon1` `icon1` BIGINT UNSIGNED NOT NULL, CHANGE `icon2` `icon2` BIGINT UNSIGNED NOT NULL, CHANGE `icon3` `icon3` BIGINT UNSIGNED NOT NULL, CHANGE `icon4` `icon4` BIGINT UNSIGNED NOT NULL, CHANGE `icon5` `icon5` BIGINT UNSIGNED NOT NULL, CHANGE `icon6` `icon6` BIGINT UNSIGNED NOT NULL, CHANGE `icon7` `icon7` BIGINT UNSIGNED NOT NULL, CHANGE `icon8` `icon8` BIGINT UNSIGNED NOT NULL; 
+ALTER TABLE `groups` CHANGE `icon1` `icon1` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon2` `icon2` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon3` `icon3` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon4` `icon4` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon5` `icon5` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon6` `icon6` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon7` `icon7` BIGINT UNSIGNED NOT NULL;
+ALTER TABLE `groups` CHANGE `icon8` `icon8` BIGINT UNSIGNED NOT NULL;
```

Comment:
o_O

Reply:
Yep - this pending sql build check is kinda weird...